src/bin/zmcontrol

```diff
@@ -52,6 +52,7 @@ my %serviceStatusList;
 my %startorder = ( 
 	"ldap" 		=> 0,
 	"zmconfigd"	=> 10,
+	"nalpeiron-daemon" => 14,
```

Comment:
@umagmrit @RakeshAMore - hide the nalpeiron name here? may be call it as `offline-daemon`?

Reply:
Its fine, we discussed this in the last weekly licensing call, there were no concerned raised for the same, for example "only office" we are following same approach.